Funcky/Extensions/HttpHeadersNonValidatedExtensions.cs

```diff
@@ -0,0 +1,13 @@
+#if HTTP_HEADERS_NON_VALIDATED
+using System.Net.Http.Headers;
+
+namespace Funcky.Extensions;
+
+public static class HttpHeadersNonValidatedExtensions
+{
+    public static Option<HeaderStringValues> GetValuesOrNone(this HttpHeadersNonValidated headers, string headerName)
+        => headers.TryGetValues(headerName, out var values)
+            ? values
+            : Option.None<HeaderStringValues>();
```

Comment:
```suggestion
            : Option<HeaderStringValues>.None;
```

Reply:
Fixed in 91c13a21e8bb89833140655bfb0c608ad7368c31